setup.py

```diff
@@ -77,6 +77,7 @@ def find_version(*file_paths):
                     "texttable>=1.6.3",
                     "scipy<=1.5.4, >=1.3.2; python_version<'3.7'",
                     "scipy>=1.3.2, <1.8; python_version>='3.7'",
+                    "numpy<1.23; python_version>='3.7'",
```

Comment:
Is there any reason for adding this dependency?

Reply:
I don't know if it worked well before, but there were some error within the scipy dependency when I tried with original one(which is installing scipy=1.7.3, numpy=1.23.1), so I added it.

error: numpy 1.23.1 is installed but numpy<1.23.0,>=1.16.5 is required by {'scipy'}

Reply:
Do you mean that `tensorflow==2.8.0` requires `scipy==1.7.3`?
I don't know why it should be included in the common depedency (torch and tf).
This is because torch has no problem with the latest develop branch: https://ci-adas-icv.iotg.sclab.intel.com/job/NNCF/job/precommit/job/ubuntu20_precommit_pytorch/53/

Reply:
Actually it was not related to tf2.8 but occurred when running common dependency with no tf or torch, 'python setup.py install', which is not overlapped by older version of numpy. Also same err occurred in latest develop branch, I think latest released numpy 1.23(June 2022) is not currently support by scipy. Maybe running install 'setup.py' without tf or torch is not common so maybe it's not a problem though..

Reply:
OK, I got it. Thanks for your explanation.